src/enclave/Enclave/ExpressionEvaluation.h

```diff
@@ -1446,6 +1609,10 @@ class FlatbuffersAggOpEvaluator {
     reset_group();
   }
 
+  size_t get_num_groups() {
```

Comment:
At the call site, "num_groups" sounds like it's referring to the number of distinct groups in the input. How about `get_num_grouping_keys()` or `get_num_grouping_exprs()`?